### PR TITLE
jenkins repo naming

### PR DESCRIPTION
**Pull request by Fguhhla**
# Description

It's needed to set a name for the jenkins repo. (At the moment the repo named is fixed 'jenkins-ci')
I add a new attribute to do this.

## Check List

- [x] All tests pass. See TESTING.md for details.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of the jenkins cookbook.
 
 ## Unreleased
 
+- Add new attribute 'repository_name' to set the name of the repository-
```
Comment (damacus) [inline CHANGELOG.md:7]:
```suggestion
- Add new attribute 'repository_name' to set the name of the repository
```

**Comment by damacus** [comment]
Can you revert the file mode change on the dokken file please?

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by damacus — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [9.5.0](https://github.com/sous-chefs/jenkins/releases/tag/9.5.0)

**Pull request merged**